app/assets/images/topic_icons/climate.svg

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I would most definitely move all icon's SVG's into a separate commit if you want to get this merged in. You're basically saying all or nothing here, and legitimate style fixes will get nixed if icons don't workout.